cli/src/commands/update-cache.js

```diff
@@ -1,44 +1,46 @@
 // @flow
 
-import {updateCacheRepo} from '../lib/libDefs';
-
-import type {Argv} from 'yargs';
 import typeof Yargs from 'yargs';
+import {
+  _setCustomCacheDir as setCustomCacheDir,
+  ensureCacheRepo,
+} from '../lib/cacheRepoUtils';
+import {path} from '../lib/node';
 
 export const name = 'update-cache';
 export const description = 'Update the flow-typed definitions cache';
 
+export type Args = {
+  cacheDir?: string,
+};
 export function setup(yargs: Yargs) {
   return yargs.usage(`$0 ${name} - ${description}`).options({
-    debug: {
```

Comment:
Why did you choose to remove debug?

Reply:
It doesn't have much sense any more because of the reason described below. The new version `ensureCacheRepo` doesn't accept verbosity parameter where `debug` parameter used to go - it rather write all required information into log all the time.